includes/wizards/class-wizard.php

```diff
@@ -45,12 +47,25 @@ abstract class Wizard {
 	 */
 	protected $menu_priority = 2;
 
+	/**
+	 * Array to store instances of section objects.
+	 *
+	 * @var Wizards\Section[]
+	 */
+	protected $sections = [];
+
 	/**
 	 * Initialize.
+	 *
+	 * @param array $args Array of optional arguments. i.e. `sections`.
```

Comment:
Nitpick: It might be worth adding some additional details for the sections entry of `$args` here to make it more clear what this should look like without needing to sift through the code in the future.

Reply:
I have added an `@example`, let me know if this is _not_ what you had in mind.

Reply:
I was thinking more along the lines of [explicitly documenting](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#docblock-formatting) (see Parameters That Are Arrays) `sections`, but this works as well!